{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2011-0775",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "pivotx/modules/module_image.php in PivotX 2.2.2 allows remote attackers to obtain sensitive information via a non-existent file in the image parameter, which reveals the installation path in an error message.  NOTE: the provenance of this information is unknown; the details are obtained solely from third party information."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "70675",
                "refsource": "OSVDB",
                "url": "http://osvdb.org/70675"
            },
            {
                "name": "43041",
                "refsource": "SECUNIA",
                "url": "http://secunia.com/advisories/43041"
            },
            {
                "name": "http://pivot-weblog.svn.sf.net/viewvc/pivot-weblog?view=revision&revision=3463",
                "refsource": "CONFIRM",
                "url": "http://pivot-weblog.svn.sf.net/viewvc/pivot-weblog?view=revision&revision=3463"
            },
            {
                "name": "pivotx-image-info-disc(64977)",
                "refsource": "XF",
                "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/64977"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}